.travis.yml

```diff
@@ -30,6 +30,11 @@ jobs:
     script:
     - "(cd website && npm install --no-package-lock)"
     - npm run lint && node .travis/check-docs.js
+  - stage: test
```

Comment:
This adds this to the existing test stage which means it will fail the build if the user agent does not match. I thought you said it will not fail the build.